src/language-js/print/member-chain.js

```diff
@@ -389,16 +399,18 @@ function printMemberChain(path, options, print) {
     printedGroups.slice(0, -1).some(willBreak) ||
     lastGroupWillBreakAndOtherCallsHaveFunctionArguments()
   ) {
-    return group(expanded);
+    result = group(expanded);
+  } else {
+    result = [
+      // We only need to check `oneLine` because if `expanded` is chosen
+      // that means that the parent group has already been broken
+      // naturally
+      willBreak(oneLine) || shouldHaveEmptyLineBeforeIndent ? breakParent : "",
+      conditionalGroup([oneLine, expanded]),
+    ];
   }
 
-  return [
-    // We only need to check `oneLine` because if `expanded` is chosen
-    // that means that the parent group has already been broken
-    // naturally
-    willBreak(oneLine) || shouldHaveEmptyLineBeforeIndent ? breakParent : "",
-    conditionalGroup([oneLine, expanded]),
-  ];
+  return label("member-chain", result);
```

Comment:
Do you think `label(result, "member-chain")` better.

Reply:
`result` can be long. On the playground, it'll be more convenient to see the string immediately. Also this order matches `join`.

Reply:
![image](https://user-images.githubusercontent.com/94334/107519701-3ce9a900-6bb9-11eb-9afa-235fc9465dee.png)